docs/content/docs/custom-resource/autoscaler.md

```diff
@@ -162,12 +162,14 @@ Setting `job.autoscaler.target.utilization.boundary: "0.2"` means that we allow
 ### Target catch-up duration and restart time
 
 When taking scaling decisions the operator need to account for the extra capacity required to catch up the backlog created during scaling operations.
-The amount of extra capacity is determined automatically by the following 2 configs:
+The amount of extra capacity is determined automatically by the following 3 configs:
```

Comment:
maybe we can remove the number simply :
` determined automatically by the following configs`

Reply:
Oh this is only a backport, then nevermind :)

Reply:
> Oh this is only a backport, then nevermind :)

Sorry, I didn't notice this comment.

If so, the 1.8 doc could remove this number, and we don't need to update the doc for main branch.